packages/datagateway-download/server/e2e-settings.json

```diff
@@ -1,18 +1,20 @@
 {
   "facilityName": "LILS",
-  "apiUrl": "http://localhost:5001",
+  "apiUrl": "http://localhost:5000",
   "downloadApiUrl": "https://localhost:8181/topcat",
   "idsUrl": "https://localhost:8181/ids",
-  "icatUrl": "https://scigateway-preprod.esc.rl.ac.uk:8181/icat",
+  "icatUrl": "https://localhost:8181/icat",
   "fileCountMax": 5000,
   "totalSizeMax": 1000000000000,
   "accessMethods": {
     "https": {
+      "_comment": "go back to this",
```

Comment:
What's this used for?

Reply:
If only JSON supported proper comments, removed in a6eae53717000e841f357e6d77c1165c99784575